samples/bluetooth/mesh/light_lc/src/lc_pwm_led.c

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
I started out that way, but i got a clutter sensation of having it there. I admit that the the pwm module is borderline small, but since the modularity level is clearly defined i would prefer to keep it as is.